lib/matplotlib/tight_layout.py

```diff
@@ -86,17 +79,11 @@ def auto_adjust_subplotpars(fig, renderer,
     else:
         hpad_inches = pad_inches
 
-    if len(subplot_list) == 0:
-        raise RuntimeError("")
-
-    if len(num1num2_list) != len(subplot_list):
-        raise RuntimeError("")
+    if len(num1num2_list) != len(subplot_list) or len(subplot_list) == 0:
+        raise ValueError
```

Comment:
I think changing the type of error is an API change, so this could do with an API note if it's going in.

Reply:
done